clients/include/testing_ger_strided_batched.hpp

```diff
@@ -191,13 +191,13 @@ void testing_ger_strided_batched(const Arguments& arg)
                                                              stride_a,
                                                              batch_count),
                               rocblas_status_success);
-
+        
         return;
     }
 
-    size_A += size_A * (batch_count - 1) + size_t(stride_a - size_A) * (batch_count - 1);
-    size_x += size_x * (batch_count - 1) + size_t(stride_x - size_x) * (batch_count - 1);
-    size_y += size_y * (batch_count - 1) + size_t(stride_y - size_y) * (batch_count - 1);
+    size_A += static_cast<size_t>(stride_a) * static_cast<size_t>(batch_count - 1);
```

Comment:
are we avoiding c style casts even for numeric type conversion?

Reply:
As explained in:

https://github.com/ROCmSoftwarePlatform/rocBLAS/blob/develop/.github/CONTRIBUTING.md

Numeric casts are preferred in function-style casts rather than C-style casts or `static_cast`s, since they are unambiguous and concise, and appear just like a constructor call.